contracts/LiquidationManager.sol

```diff
@@ -11,12 +14,121 @@ import { ExponentialNoError } from "./Utils/ExponentialNoError.sol";
  * @author Venus
  * @notice This contract is designed to be used in conjunction with the venus protocol's liquidation process.
  */
-contract LiquidationManager is ExponentialNoError {
+contract LiquidationManager is AccessControlledV8, ExponentialNoError {
+    /// @notice Comptroller address, used to interact with the protocol's core functionalities
+    ComptrollerInterface public immutable comptroller;
```

Comment:
I would not include here the `comptroller` reference, "only" to check if the market is listed or not. I would remove it to reduce the inter-dependencies